run_tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Run unit tests
+printf "\n--------------------- Unit Tests ----------------------\n"
+py.test inspirehep tests/unit
+unit_tests_exit_code=$?
+
+# Make documentation
+printf "\n\n--------------- Creating Documentation ----------------\n"
+make -C docs html
+create_docs_exit_code=$?
+
+if [ unit_tests_exit_code != 0 ]
```

Comment:
```bash
if [[ "$unit_tests_exit_code" != "0" ]]; then
```
* `[[]]` instead of `[]` because `[]` is acually a link to the binary `/usr/bin/[`, that is an external command to bash, and undergoes all the bash expansion and spaces stuff with it's issues, while `[[]]` is a builtin command, and has more sane syntax as it does not undergo the same expansion step (it can be avoided though using proper quoting, that is sometimes tricky):
```shell
dcaro@pcrcssis001$ a='one two'

dcaro@pcrcssis001$ [ $a == 'one two' ] && echo yes || echo no
bash: [: too many arguments
no

dcaro@pcrcssis001$ [[ $a == 'one two' ]] && echo yes || echo no
yes

dcaro@pcrcssis001$ a='*'

dcaro@pcrcssis001$ [[ $a == '*' ]] && echo yes || echo no
yes

dcaro@pcrcssis001$ [ $a == '*' ] && echo yes || echo no
bash: [: too many arguments
no

dcaro@pcrcssis001$ echo $a  # when there are no quotes, bash resolves $a to * and then expands it
...
A list with all your files
...

dcaro@pcrcssis001$ echo "$a"  # with quotes, bash does not expand the * after resolving the var
*